Source/WebCore/platform/graphics/MediaPlayer.h

```diff
@@ -411,7 +411,7 @@ class WEBCORE_EXPORT MediaPlayer : public MediaPlayerEnums, public ThreadSafeRef
     LayerHostingContextID hostingContextID() const;
     FloatSize videoLayerSize() const;
     void videoLayerSizeDidChange(const FloatSize&);
-    void setVideoLayerSizeFenced(const FloatSize&, WTF::MachSendRight&&);
+    void setVideoLayerSizeFenced(const FloatSize&, WTF::MachSendRight&&, Vector<uint8_t>&&);
```

Comment:
Can we use a `LayerHostingInformation` instead of a separate MachSendRight and vector everywhere?